### ISSUE TITLE
imageDescriptor ref page error for 1.2 pages

### ISSUE THREAD

**Issue by oddhack** [KhronosGroup/OpenCL-Registry#16]
Moved from internal issue 10654:

The cl_mem_object_type image_type is missing from the online documentation at:

http://www.khronos.org/registry/cl/sdk/1.2/docs/man/xhtml/imageDescriptor.html

section 5.3.1.2 specifies an image descriptor with the fields:

```
typedef struct _cl_image_desc {
   cl_mem_object_type image_type,
   size_t image_width;
   size_t image_height;
   size_t image_depth;
   ....
} cl_image_desc;
```

just looks like a cut/paste error.

**Comment by Oblomov** [comment]
To be specific, the page does mention the possible values that `image_type` can hold. What it does not mention is that cl_mem_object_type is just a typedef for `cl_uint`.

**Comment by oddhack** [comment]
Closed by #18
